### PR TITLE
Reduce metrics noise + fix some metrics reporting  bugs

### PR DESCRIPTION
**Pull request by PavelZaytsev**
## Overview

* Remove 95 percentile from logs.
* Don't log zero values.
* Fix some timing bugs.

### CODE REVIEW

**Review by Maithem — APPROVED** [review]

**Review by tomato-coding — APPROVED** [review]
looks good

File: common/src/main/java/org/corfudb/common/metrics/micrometer/registries/LoggingMeterRegistryWithHistogramSupport.java [hunk 1]
```diff
@@ -65,22 +65,22 @@ public LoggingMeterRegistryWithHistogramSupport(LoggingRegistryConfig config,
...
-        if (Double.isFinite(count)) {
+        if (Double.isFinite(count) && count != 0) {
             return Stream.of(influxLineProtocol(id, "counter", Stream.of(new Field("value", count))));
         }
         return Stream.empty();
     }
 
     Stream<String> writeGauge(Meter.Id id, Double value) {
-        if (Double.isFinite(value)) {
+        if (Double.isFinite(value) && value != 0) {
```
Comment (hisundar) [inline common/src/main/java/org/corfudb/common/metrics/micrometer/registries/LoggingMeterRegistryWithHistogramSupport.java:75]: Guage can go up and down right, so is this ok for printing only non-zero value?

**Review by hisundar — APPROVED** [review]
Just a minor question about Guages, but otherwise this change looks good to me, thanks

File: common/src/main/java/org/corfudb/common/metrics/micrometer/registries/LoggingMeterRegistryWithHistogramSupport.java (hunk 1 above)
Comment (PavelZaytsev) [reply common/src/main/java/org/corfudb/common/metrics/micrometer/registries/LoggingMeterRegistryWithHistogramSupport.java]: I agree, unlike every other meter (timer, distribution), gauges report current values and not rates. 0 is a valid current value to be reported.

**Pull request merged**